Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:0.10.36
+FROM node:7.3
```

Comment:
No.. It throws this error:

```
/usr/local/lib/node_modules/openshift-auth-proxy/node_modules/lru_map/lru.js:78
  let entry, limit = this.limit || Number.MAX_VALUE;
  ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/openshift-auth-proxy/lib/authHandlers.js:10:22)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

I will see how to change this module if it is not possible to accommodate this.